tests/testSchema.fbs

```diff
@@ -1,6 +1,16 @@
 
 namespace test;
 
+enum TestEnum : uint {
```

Comment:
It applies the standard schema from FlatBuffers, we also discussed but no conclusion, we can assume in our parser it is uint.